lbrynet/reflector/server/server.py

```diff
@@ -11,27 +11,89 @@
 
 log = logging.getLogger(__name__)
 
+SEND_SD_BLOB = 'send_sd_blob'
+SEND_BLOB = 'send_blob'
+RECEIVED_SD_BLOB = 'received_sd_blob'
+RECEIVED_BLOB = 'received_blob'
+NEEDED_BLOBS = 'needed_blobs'
+VERSION = 'version'
+BLOB_SIZE = 'blob_size'
+BLOB_HASH = 'blob_hash'
+SD_BLOB_SIZE = 'sd_blob_size'
+SD_BLOB_HASH = 'sd_blob_hash'
+
 
 class ReflectorServer(Protocol):
     def connectionMade(self):
         peer_info = self.transport.getPeer()
         log.debug('Connection made to %s', peer_info)
         self.peer = self.factory.peer_manager.get_peer(peer_info.host, peer_info.port)
         self.blob_manager = self.factory.blob_manager
+        self.protocol_version = self.factory.protocol_version
         self.received_handshake = False
-        self.received_descriptor = False
-        self.protocol_version = REFLECTOR_V2
         self.peer_version = None
         self.receiving_blob = False
         self.incoming_blob = None
         self.blob_write = None
-        self.blob_finished_d = None
+        self.blob_f_d = None
         self.cancel_write = None
         self.request_buff = ""
 
     def connectionLost(self, reason=failure.Failure(error.ConnectionDone())):
         log.info("Reflector upload from %s finished" % self.peer.host)
 
+    def handle_error(self, err):
+        log.error(err.getTraceback())
+        self.transport.loseConnection()
+
+    def send_response(self, response_dict):
+        self.transport.write(json.dumps(response_dict))
+
+    ############################
+    # Incoming blob file stuff #
+    ############################
+
+    def clean_up_failed_upload(self, err, blob):
+        log.warning("Failed to receive %s", blob.blob_hash[:16])
+        if err.check(DownloadCanceledError):
+            err.trap(DownloadCanceledError)
+            self.blob_manager.delete_blobs([blob.blob_hash])
+        else:
+            log.exception(err)
+        return False
+
+    def log_received_blob(self, blob):
+        log.info("Received %s", blob.blob_hash[:16])
```

Comment:
Since the __repr__ and __str__ functions of blob already do this (https://github.com/lbryio/lbry/blob/master/lbrynet/core/HashBlob.py#L201)

This should just be log.info('Received %s", blob).

And, think about it - what if we decide that we do want the full hash in the logs?  I don't want to have to change the code in multiple different places.